src/registrar/templates/django/admin/domain_change_form.html

```diff
@@ -1,7 +1,14 @@
 {% extends 'admin/change_form.html' %}
+{% load i18n static %}
+
+{% block extrahead %}
+{{ block.super }}
+<script type="text/javascript" src="{% static 'js/get-gov-admin.js' %}" defer></script>
```

Comment:
This has to do with the structure of the template... Basically this js file is what allows it to open in another tab. There is not really a method in Django to redirect to another tab as I believe that is handled by the browser.

All of the 'buttons' on this page (mostly placehold) are nested in a shared form object. Without it, I'd basically need to do a somewhat extensive form + django rewrite which I'd like to avoid doing if possible as I do not know if that will break things. That said - I think we may be able to avoid this js implementation (potentially in another PR given that it looks like we're adding more buttons to this page in general), what are you thoughts on that?